pkg/apis/operator/v1/types.go

```diff
@@ -206,6 +214,12 @@ type CalicoNetworkSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum=None;Multus
 	MultiInterfaceMode *MultiInterfaceMode `json:"multiInterfaceMode,omitempty"`
+
+	// ContainerIpForwarding configures whether ip forwarding will be enabled for containers in the CNI configuration.
+	// Default: Disabled
+	// +optional
+	// +kubebuilder:validation:Enum=Enabled;Disabled
+	ContainerIpForwarding *ContainerIpForwardingType `json:"containerIpForwarding,omitempty"`
```

Comment:
Should be `containerIPForwarding` not `containerIpForwarding` to match conventional k8s API casing

Reply:
Same for both the golang field name and the json tag

Reply:
done